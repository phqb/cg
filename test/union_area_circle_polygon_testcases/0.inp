1 2 3
4
-2.18 5.26
4.12 4.16
4.98 2.58
-2.76 2.36