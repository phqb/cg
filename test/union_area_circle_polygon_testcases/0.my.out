9.3965875123